src/VisualStudio/Core/Impl/Options/Style/EnumCodeStyleOptionViewModel.cs

```diff
@@ -42,18 +58,30 @@ static EnumCodeStyleOptionViewModel()
             OptionSet options,
             string groupName,
             List<CodeStylePreference> preferences)
+            : this(option, language: null, description, enumValues, previews, info,
+                   options, groupName, preferences)
+        {
+        }
+
+        private EnumCodeStyleOptionViewModel(
+            IOption option,
+            string language,
```

Comment:
Perhaps make `language` an optional parameter instead.
Also, nit: unnecessary empty line at line 49

Reply:
i woudl prefer it be required, so that callers have to think about this is a language specific preference or not.